client/ayon_hiero/api/plugin.py

```diff
@@ -648,17 +664,39 @@ class PublishClip:
     rename_default = False
     hierarchy_default = "{_folder_}/{_sequence_}/{_track_}"
     clip_name_default = "shot_{_trackIndex_:0>3}_{_clipIndex_:0>4}"
-    base_product_name_default = "<track_name>"
+    subset_name_default = "<track_name>"
```

Comment:
`subset` is OpenPype term, should be `product`.

Reply:
I've renamed subset for product everywhere.